forwardbot_gitlab/api.py

```diff
@@ -0,0 +1,67 @@
+from typing import TYPE_CHECKING, Dict
+from urllib.parse import quote, urljoin
+
+from requests import Request, Response, Session
+
+if TYPE_CHECKING:
+    from .models.merge_request import MergeRequest
+
+
+def get_api_data(env):
+    config = env["ir.config_parameter"].sudo()
+    token = config.get_param("forwardbot_gitlab.token")
```

Comment:
IMHO the token should be configured for each runbot.repo

Similar to current token

Since that each repo could be used from different gitlab server

e.g. gitlab.com, git.vauxoo.com, git.islamic, git.other and so on

Same with base_url, it could be obtained from the runbot.repo name directly

Reply:
I just realized the runbot.repo doesn't exists here since that this module doesn't depends on runbot modules

So, I need to re-check my last review with this in mind